### PR TITLE
Added advertisement classes and enhanced images

### PR DESCRIPTION
**Pull request by gitesh-webkul**
## Issue Reference
#7190

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@gitesh-webkul 
Resolve the **conflicts**.

**Comment by Sanjaybhattwebkul** [comment]
@gitesh-webkul 
resolve the **conflicts**

File: packages/Webkul/Velocity/src/Resources/views/shop/home/advertisements/advertisement-four.blade.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ class="{{ $isLazyLoad ? 'lazyload' : '' }}"
                     @endif
                 </div>
 
-                <div class="col-lg-4 col-12 advertisement-container-block offers-ct-panel">
+                <div class="advertisement-container-block adv2">
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/home/advertisements/advertisement-four.blade.php:40]: Use a meaningful name like the first one is `advertisement-container-block` then the remaining classes should follow the convention like `advertisement-1` or may be try to handle without numbering like `advertisement-container-block advertisement-small advertisement-large advertisement-medium`  whatever is suitable.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@gitesh-webkul 
**Still not fixed**, the images are getting **stretched** in **mobile** view on **home page.**, Please find the screenshot [Here](https://webkul.chatwhizz.com/share/screenshot/63f4ab0faac9534fac264f36)

**Review by Sanjaybhattwebkul — DISMISSED** [review]

**Review by devansh-webkul — DISMISSED** [review]

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Pull request merged**